@xen-orchestra/web-core/lib/locales/fr.json

```diff
@@ -1,89 +1,78 @@
+    "core.group": "Grouper",
+    "core.hide": "Masquer",
+    "core.open": "Ouvrir",
+    "core.quick-actions": "Actions rapides",
+    "core.search": "Rechercher",
+    "core.query-search-bar.label": "Moteur de recherche",
+    "core.query-search-bar.placeholder": "Écrivez votre requête…",
+    "core.query-search-bar.use-query-builder": "Utiliser le constructeur de requête",
+    "core.select.all": "Tout sélectionner",
+    "core.select.none": "Tout désélectionner",
+    "core.select.unselect": "Tout désélectionner",
+    "core.select.n-selected-of": "{count} objet sélectionné sur {total} | {count} objet sélectionné sur {total} | {count} objets sélectionnés sur {total}",
+    "core.sidebar.close": "Fermer la barre latérale",
+    "core.sidebar.lock": "Verrouiller la barre latérale",
+    "core.sidebar.open": "Ouvrir la barre latérale",
+    "core.sidebar.unlock": "Déverrouiller la barre latérale",
+    "core.sort.ascending": "Trier par ordre croissant",
+    "core.sort.descending": "Trier par ordre décroissant",
+    "dark-mode.enable": "Activer le mode sombre",
+    "dark-mode.disable": "Désactiver le mode sombre",
+    "dark-mode.auto": "Mode sombre automatique",
+    "access-forum": "Accès au forum",
+    "dashboard": "Tableau de bord",
+    "documentation-name": "Documentation {name}",
+    "error-no-data": "Erreur, impossible de collecter les données.",
+    "exit-fullscreen": "Quitter le plein écran",
+    "fullscreen": "Plein écran",
+    "hosts": "Hôtes",
+    "learn-more": "En savoir plus",
+    "loading-in-progress": "Chargement en cours…",
+    "log-out": "Se déconnecter",
+    "master": "Hôte primaire",
+    "n-vms": "1 VM | {n} VMs",
+    "network": "Réseau",
+    "no-data": "Aucune donnée",
+    "object-not-found": "L'objet {id} est introuvable…",
+    "open-console-in-new-tab": "Ouvrir la console dans un nouvel onglet",
+    "other": "Autre",
+    "page-not-found": "Cette page est introuvable…",
+    "patches": "Patches",
+    "power-on-vm-for-console": "Allumez votre VM pour accéder à sa console",
+    "power-on-host-for-console": "Allumez votre hôte pour accéder à sa console",
+    "professional-support": "Support professionel",
+    "receive": "Recevoir",
+    "running-vm": "VM en cours d'exécution | VMs en cours d'exécution",
+    "see-all": "Voir tout",
+    "select-to-see-details": "Sélectionnez un élement pour voir les details",
+    "send": "Envoyer",
+    "send-ctrl-alt-del": "Envoyer Ctrl+Alt+Suppr",
+    "stats": "Stats",
+    "storage": "Stockage",
+    "system": "Système",
+    "tasks": "Tâches",
+    "tasks.n-subtasks": "{n} sous-tâche | {n} sous-tâches",
+    "tasks.no-tasks": "Aucune tâche",
+    "tasks.quick-view": "Vue rapide des tâches",
+    "tasks.quick-view.all": "Toutes",
+    "tasks.quick-view.done": "Terminées",
+    "tasks.quick-view.failed": "Échouées",
+    "tasks.quick-view.in-progress": "En cours",
+    "total": "Total",
+    "total-cpus": "Total CPUs",
+    "vms": "VMs"
```

Comment:
Any way to make it respect our formatting? (indenting with 2 spaces)

Reply:
Yes, I've seen an option in Weblate to configure it and sort the keys alphabetically.
I will try to tweak some other things, like commit messages etc.